engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/K8sDeploymentManager.scala

```diff
@@ -187,7 +187,8 @@ class K8sDeploymentManager(modelData: BaseModelData, config: K8sDeploymentManage
     //We wait for deployment removal before removing configmaps,
     //in case of crash it's better to have unnecessary configmaps than deployments without config
     for {
-      services <- k8s.deleteAllSelected[ListResource[Service]](selector)
+      services <- k8s.listSelected[ListResource[Service]](selector)
```

Comment:
Can you add comment that it is split into two steps because of missing k8s feature in version xxx ?